### PR TITLE
Add dynamic routing test cases

### PR DESCRIPTION
**Pull request by asmorodskyi**

### CODE REVIEW

**Comment by asmorodskyi** [comment]
The work is still in progress, but because I am doing some things which can raise a lot of discussions I decided to already open PR.  
@ldevulder  - please take a look into my idea with "stealing" your wait_support_server module . Is it ok for you ? 
@pevik you asked me to share PR

**Comment by pevik** [comment]
@asmorodskyi Thx.

File: tests/support_server/wait_support_server.pm [hunk 1]
```diff
@@ -7,8 +7,8 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
-# Summary: Start HA support server and check network connectivity
-# Maintainer: Loic Devulder <ldevulder@suse.com>
+# Summary: Start support server and check network connectivity
```
Comment (ldevulder) [inline tests/support_server/wait_support_server.pm]: My summary seems a little bit old, as there is no network connectivity check in it...

A better summary could be simply: "Wait for support server".

**Comment by ldevulder** [comment]
@asmorodskyi, ok for me :) I just asked you to change the summary, mine was not so good.

**Comment by asmorodskyi** [comment]
ready to merge .
Proof runs :
http://kimball.arch.suse.de/tests/620
http://kimball.arch.suse.de/tests/621

**Comment by DimStar77** [comment]
Fails on o3/opensuse: https://openqa.opensuse.org/tests/611813#step/basic/63

**Pull request merged**